workers/base_task.py

```diff
@@ -86,7 +87,14 @@ def run(self, prev_result=None, **params):
         # results can also be part of the params array in some cases
         if 'result' in params:
             self._add_prev_result_to_results(params['result'])
-        return self._merge_result(self.execute_task())
+
+        try:
+            task_result = self.execute_task()
+        except:  # noqa: ignore bare except
```

Comment:
Falls sich wer über den Kommentar wundert, das ist speziell für flake8 und hindert ihn am meckern über den base except, den ich in diesem Spezielfall für zulässig halte.

Reply:
Geht so nicht die Fehlermeldung verloren? Also wie würde ich als User oder Admin herausfinden, was genau schief gelaufen ist?

Die Error-Message sollte mindestens geloggt werden, besser fände ich es, wenn sie sogar in der Job-DB gespeichert würde, damit man sie über die API abfragen und im Frontend darstellen kann.